api/api-doc/storage_service.json

```diff
@@ -1105,6 +1105,14 @@
                      "allowMultiple":false,
                      "type":"string",
                      "paramType":"query"
+                  },
+                  {
+                     "name":"ignore_nodes",
+                     "description":"List of dead nodes to ingore in removenode operation",
+                     "required":false,
+                     "allowMultiple":false,
+                     "type":"string",
```

Comment:
Shouldn't the type be array?

Reply:
@amnonh please advise

Reply:
@avikivity It's a query parameter, we don't support arrays in query parameters

Reply:
What about allowMultiple?

Reply:
@amnonh ping here